app/Console/Commands/PollTwitter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Log;
```

Comment:
Your question originally here was:

> I didn't know that was possible - did Laravel always used to work like that?

I never actually need to worry about it until 5.0, so I can only speak to it working that way back to 5.0, but I dont' know of any significant changes to the structure of Façades for quite a while, so I would assume yes?